packages/functional-tests/tests/postVerify/accountRecovery.spec.ts

```diff
@@ -0,0 +1,129 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+const password = 'passwordzxcv';
+let key;
+let email;
+
+test.describe('post verify - account recovery', () => {
+  test.beforeEach(async ({ target, pages: { login } }) => {
+    // Generating and consuming recovery keys is a slow process
+    test.slow();
+    email = login.createEmail();
+    await target.auth.signUp(email, password, {
+      lang: 'en',
+      preVerified: 'true',
+    });
+    await login.clearCache();
+  });
+
+  test.afterEach(async ({ target }) => {
+    if (email) {
+      // Cleanup any accounts created during the test
+      try {
+        await target.auth.accountDestroy(email, password);
+      } catch (e) {}
+    }
+  });
+
+  test('create account recovery', async ({
+    target,
+    pages: { page, login, postVerify },
+  }) => {
+    await page.goto(target.contentServerUrl, {
+      waitUntil: 'load',
+    });
+    await login.fillOutEmailFirstSignIn(email, password);
+
+    //Verify logged in on Settings page
+    expect(await login.loginHeader()).toBe(true);
+
+    await page.goto(
+      `${target.contentServerUrl}/post_verify/account_recovery/add_recovery_key`,
+      { waitUntil: 'networkidle' }
+    );
+
+    //Verify account recovery header
+    expect(await postVerify.isAccountRecoveryHeader()).toBe(true);
+
+    //Add recovery key
+    await postVerify.addRecoveryKey(password);
+    await postVerify.submit();
+
+    // Store key to be used later
+    key = await postVerify.getKey();
+    await postVerify.clickDone();
+
+    //Enter invalid key
+    await postVerify.inputRecoveryKey('invalid key');
+
+    //Verify error message
+    expect(await postVerify.getTooltipError()).toContain(
+      'Invalid account recovery key'
+    );
+
+    //Enter correct key
+    await postVerify.inputRecoveryKey(key);
+    await page.waitForNavigation();
+
+    //Verify post verify account recovery key complete header
+    expect(await postVerify.isAccountRecoveryVerifiedHeader()).toBe(true);
+  });
+
+  test('abort account recovery at add_recovery_key', async ({
+    target,
+    pages: { page, login, postVerify },
+  }) => {
+    await page.goto(target.contentServerUrl, {
+      waitUntil: 'load',
+    });
+    await login.fillOutEmailFirstSignIn(email, password);
+
+    //Verify logged in on Settings page
+    expect(await login.loginHeader()).toBe(true);
```

Comment:
I like this comment. I feel like name `login.loginHeader()` isn't super clear. If I were to read that without any context, I'd assume it's a header that is visible when a user is asked to login. I would not assume it's the header on the settings page that shows up after the user has succesfully loggedIn. Probably a change for separate PR, but renaming this might be good for long term maintainability.

Reply:
Sure, I will change this but since it's used in a lot of places I will create a separate PR to address this.

Reply:
@ashrivastava-qa I don't think we actually need post verify account recovery tests. 😭 We just cancelled the epic for the React conversion, [see here](https://mozilla-hub.atlassian.net/browse/FXA-6510?focusedCommentId=680801). Similarly, we cancelled post verify secondary email views.

Reply:
let's add this for now since it's already done. Once all the pages have been migrated to react we can remove this.